pkg/plugins/plugins.go

```diff
@@ -252,21 +266,62 @@ func (scanner *PluginScanner) loadPluginJson(pluginJsonFilePath string) error {
 	}
 
 	if pluginCommon.Id == "" || pluginCommon.Type == "" {
-		return errors.New("Did not find type and id property in plugin.json")
+		return errors.New("did not find type or id properties in plugin.json")
+	}
+
+	if pluginCommon.Id == "gel" {
```

Comment:
Think better to check type in case we rename gel and the feature toggle is basically for transform plugin type.

```suggestion
	if pluginCommon.Type == "transform" {
```

Reply:
Good point!

Reply:
I think there should be some comment about why as well. Otherwise, this looks good!